### PR TITLE
Changing INVESTOR_RATIO to be defined at construct time

### PR DESCRIPTION
**Pull request by ggviana**

### CODE REVIEW

File: contracts/vaults/STETHVault.sol [hunk 1]
```diff
@@ -53,6 +53,9 @@ contract STETHVault is BaseVault {
             string(abi.encodePacked(_asset.symbol(), "vv"))
         )
     {
+        // The precision of this number is set by the variable DENOMINATOR. 5000 is equivalent to 50%
+        INVESTOR_RATIO = _investorRatio;
```
Comment (aviggiano) [inline contracts/vaults/STETHVault.sol:61]: Check that `INVESTOR_RATIO <= DENOMINATOR`

File: contracts/vaults/STETHVault.sol [hunk 2]
```diff
@@ -133,6 +140,8 @@ contract STETHVault is BaseVault {
         if (supply != 0) {
             if (totalAssets() >= lastRoundAssets) {
                 roundAccruedInterest = totalAssets() - lastRoundAssets;
+                // Notice that `investmentAmount` will be truncated to 0 if
```
Comment (aviggiano) [inline contracts/vaults/STETHVault.sol:143]: This scenario could be tested with a unit test

Comment (aviggiano) [reply contracts/vaults/STETHVault.sol:143]: Also, now that `INVESTOR_RATIO` is a parameter, other edge case values could also be tested in addition to `DENOMINATOR + 1`:

- `0`
- `DENOMINATOR`

**Review by Robsonsjre — APPROVED** [review]
